package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java

```diff
@@ -378,7 +324,17 @@ enum Option {
 	VCD_IMPORT_OVERWRITE_MODE(
 			"vcd_import_overwrite_mode",
 			ConfigurationVcd.PACKAGE_IMPORT_OVERWRITE_MODE),
-
+	/**
+	 * Configurations.
+	 *
+	 * VRA server.
+	 *
+	 * Yes, this is still for `vra` which is deprecated. That being said it's being
+	 * used for some reason in the Installer
+	 */
+	VRA_SERVER(
```

Comment:
Which is the part of installer which uses this? is it something like a base property that vra ng uses as well? Is there any way to completely remove this?

Reply:
```java
	private static void readVroProperties(final Input input, final boolean hasVraNgPackages) {
		input.getText().getTextTerminal().println("vRealize Orchestrator Configuration:");
		userInput(input, Option.VRO_SERVER, "  vRO FQDN", input.get(Option.VRA_SERVER));
		Validate.host(input.get(Option.VRO_SERVER), input.getText());

```

realistically it's just the namig scheme, but .... I don't want to touch this now, seems ok to leave like this